src/main/java/mate/academy/spring/service/RoleService.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.spring.service;
+
+import mate.academy.spring.model.Role;
+
+public interface RoleService {
+    Role add(Role role);
+
+    Role getByName(Role.RoleName roleName);
```

Comment:
Fix parameter that you should pass to the service 
![image](https://user-images.githubusercontent.com/32339836/236505826-65145a79-8467-48d6-a073-370a68dbd1de.png)